src/nft/components/explore/CarouselCard.tsx

```diff
@@ -164,15 +165,20 @@ const TableElement = styled.div`
 
 interface MarketplaceRowProps {
   marketplace: string
-  floorInEth?: string
-  listings?: string
+  floorInEth?: number
+  listings?: number
 }
 
 export const MarketplaceRow = ({ marketplace, floorInEth, listings }: MarketplaceRowProps) => {
   return (
     <>
       <TableElement>{marketplace}</TableElement>
-      <TableElement>{floorInEth ?? '-'} ETH</TableElement>
+      <TableElement>
+        {typeof floorInEth === 'number'
```

Comment:
Wouldn't a simpler check just be `floorInEth !== undefined`?

Reply:
Yeah, that works too. Updated. Initially, I looked at the direct replacement of `??` but should've just looked up Typescript type to realize it can be either undefined or a number.